cpp/log/sqlite_db-inl.h

```diff
@@ -17,8 +17,21 @@
 // new node, it would do an initial load of its local database with
 // "synchronous" set to OFF, then put it back before starting normal
 // operation.
-DEFINE_bool(sqlite_synchronous_off, false,
-            "whether to set SQLite's \"synchronous\" to OFF");
+DEFINE_string(sqlite_synchronous_mode, "FULL",
+              "While SQLite synchronous option to use");
```

Comment:
Me fail English? That's unpossible!

Options are documented here: https://www.sqlite.org/pragma.html

I've added a comment and a hint in the flags' help strings